examples/47a_ampere_gemm_universal_streamk_residual/ampere_gemm_universal_streamk_residual.cu

```diff
@@ -0,0 +1,703 @@
+    {                                         // epilogue parameters
+      ElementAccumulator(options.alpha),
+      ElementAccumulator(options.beta)
+    },
+    tensor_a.device_data(),                   // ptr_A
+    tensor_b.device_data(),                   // ptr_B
+    tensor_c.device_data(),                   // ptr_C
+    tensor_d.device_data(),                   // ptr_D
+    tensor_z.device_data(),                   // ptr_Vector
+    tensor_t.device_data(),                   // ptr_Tensor
+    options.problem_size.mk().product(),      // batch_stride_A
+    options.problem_size.nk().product(),      // batch_stride_B
+    options.problem_size.mn().product(),      // batch_stride_C
+    options.problem_size.mn().product(),      // batch_stride_D
+    options.problem_size.mn().product(),      // batch_stride_Vector
+    options.problem_size.mn().product(),      // batch_stride_Tensor
+    tensor_a.layout().stride(0),              // stride_a
+    tensor_b.layout().stride(0),              // stride_b
+    tensor_c.layout().stride(0),              // stride_c
+    tensor_d.layout().stride(0),              // stride_d
+    tensor_z.layout().stride(0),              // stride_Vector
+    tensor_t.layout().stride(0),              // stride_Tensor
+    options.avail_sms);                       // avail_sms
+}
+
+
+/// Tensor MSE function
+//// Useful when we expect outputs to approximately match, but 
+//// not exactly.
+template <
+  typename Element,               ///< Element type
+  typename Layout>                ///< Layout function
+struct TensorMSEFunc {
+
+  //
+  // Data members
+  //
+
+  cutlass::TensorView<Element, Layout> lhs;
+  cutlass::TensorView<Element, Layout> rhs;
+  double err;
+  double cnt;
+
+  /// Ctor
+  TensorMSEFunc(): err(0), cnt(0) { }
+
+  /// Ctor
+  TensorMSEFunc(
+    cutlass::TensorView<Element, Layout> const &lhs_,
+    cutlass::TensorView<Element, Layout> const &rhs_
+  ) :
+    lhs(lhs_), rhs(rhs_), err(0), cnt(0) { }
+
+  /// Visits a coordinate
+  void operator()(cutlass::Coord<Layout::kRank> const &coord) {
+
+    Element lhs_ = lhs.at(coord);
+    Element rhs_ = rhs.at(coord);
+    
+    cnt += double(1.0);
+    double diff = double(lhs_) - double(rhs_);
+    err += double(diff * diff);
+  }
+
+  double get_err() const {
+    if (cnt == double(0.0))
+      return double(0.0);
+    return err / cnt;
+  }
+};
+
+/// Mean squared error
+template <
+  typename Element,               ///< Element type
+  typename Layout>                ///< Layout function
+double TensorMSE(
+  cutlass::TensorView<Element, Layout> const &lhs, 
+  cutlass::TensorView<Element, Layout> const &rhs) {
+
+  // Extents must be identical
+  if (lhs.extent() != rhs.extent()) {
+    return double(-1);
+  }
+
+  TensorMSEFunc<Element, Layout> func(lhs, rhs);
+  cutlass::reference::host::TensorForEach(
+    lhs.extent(),
+    func
+  );
+
+  return func.get_err();
+}
```

Comment:
We could probably use the existing `TensorRelativeErrorMetric` (defined [here](https://github.com/NVIDIA/cutlass/blob/66d9cddc832c1cdc2b30a8755274f7f74640cfe6/tools/util/include/cutlass/util/reference/host/error_metrics.h#L51)) instead. This metric is used in existing CUTLASS examples that test for numerical accuracy (see the 3xTF32 example [here](https://github.com/NVIDIA/cutlass/blob/77549ae6c8cf31c7ac4c8b88180a8708a8683da4/examples/27_ampere_3xtf32_fast_accurate_tensorop_gemm/27_ampere_3xtf32_fast_accurate_tensorop_gemm.cu#L633-L638)).